guides/infra/spk-infra-generation-pipeline.md

```diff
@@ -69,13 +71,12 @@ AZDO_PROJECT_NAME: The name of the project in your Azure DevOps organization whe
 
 You can use `spk` to create the Azure DevOps Variable Groups by executing
 `spk variable-group create` described in this
-[doc](../../guides/variable-group.md).
-This will require you to create a variable group manifest similar to the
-following:
+[doc](../../guides/variable-group.md). This will require you to create a
+variable group manifest similar to the following:
 
-```
+```yml
 name: "spk-infra-hld-vg"
-description: "variable groupd for infra hld"
+description: "variable group for infra hld"
 type: "Vsts"
 variables:
   ACCESS_TOKEN_SECRET:
```

Comment:
I think `"<SP-TENANT-ID>`  is missing a trailing '"'.

Reply:
Added.